taskRequests/details/style.css

```diff
@@ -261,3 +324,53 @@ body {
     max-width: 40%;
   }
 }
+
+.requestor_details_modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
```

Comment:
z-index should also be kept in variables, as it helps when adding z-index to other elements to easily find the issue happening sometimes

Reply:
Done